CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+0.20.1 / TBA
+===================
+- Display actual search term when no matches in results.
```

Comment:
This confused me a bit as it isn't when there are no (pharmacy) results but when there are no place/postcode results. How about `Display actual search term when no place or postcodes match`?